src/documentation/pages/components/icon/examples/_icons.vue

```diff
@@ -0,0 +1,18 @@
tion', 'box-cloudformation', 'box-container', 'box-deployment', 'box-fill', 'box-outline', 'box-script', 'box', 'cart-alt', 'cart-outline', 'cart', 'chart-bar', 'chart-line', 'chart-pie', 'chat-outline', 'chat', 'check-alt', 'check', 'checkbox-checked', 'checkbox', 'chevron-down', 'chevron-left', 'chevron-right', 'chevron-up', 'circle-alert-outline', 'circle-alert', 'circle-arrow-down-outline', 'circle-arrow-down', 'circle-arrow-up-outline', 'circle-arrow-up', 'circle-check-outline', 'circle-check', 'circle-clock-outline', 'circle-clock', 'circle-code-outline', 'circle-code', 'circle-gear-outline', 'circle-gear', 'circle-info-outline', 'circle-info', 'circle-minus-outline', 'circle-minus', 'circle-outline', 'circle-play-outline', 'circle-play', 'circle-plus-outline', 'circle-plus', 'circle-question-outline', 'circle-question', 'circle-reload-outline', 'circle-reload', 'circle-slash', 'circle-slash-outline', 'circle-stop-outline', 'circle-stop', 'circle-user', 'circle-warning', 'circle-wrench', 'circle-x-outline', 'circle-x', 'circle', 'cloud', 'comment-dots-outline', 'comment-dots', 'comment-outline', 'comment', 'comments-dots-outline', 'comments-dots', 'comments-outline', 'comments', 'compose-cancel', 'compose', 'copy', 'cost', 'database-outline', 'database', 'date', 'date-alt', 'delete', 'edge-solutions', 'edit', 'edit-cancel', 'edit-create', 'external-link', 'file-error-outline', 'file-export', 'file-invoice', 'file-outline', 'file', 'filter', 'flag', 'folder-gear', 'folder-open', 'folder', 'globe-network', 'globe-outline', 'grid', 'hand-gear', 'handshake', 'heart', 'home-outline', 'home', 'inbox', 'inbox-multiple', 'key', 'life-ring', 'link', 'list', 'list-alt', 'lock-outline', 'lock', 'login', 'logo-amazon', 'logo-aws', 'logo-azure', 'logo-centurylink', 'logo-cloudstack', 'logo-dimensiondata', 'logo-facebook', 'logo-github', 'logo-google', 'logo-hp', 'logo-linkedin', 'logo-linux', 'logo-openstack', 'logo-rackspace', 'logo-slack', 'logo-twitter', 'logo-vcloud', 'logo-vsphere', 'logo-youtube', 'logout', 'mail-outline', 'mail', 'map-marker-alt', 'map-marker', 'map-marker-outline', 'menu', 'microphone-mute', 'microphone', 'minus-alt', 'minus', 'more-vert', 'more', 'nodes', 'number', 'page-first', 'page-last', 'paste', 'pause', 'phone', 'play', 'plus-alt', 'plus', 'print', 'radio-checked', 'radio', 'refresh', 'reload', 'reset', 'rocket', 'search', 'settings', 'share', 'shield', 'spinner', 'square-minus-outline', 'square-minus', 'square-plus-outline', 'square-plus', 'star-outline', 'star', 'stop', 'sync', 'table-column-settings', 'table', 'tag-outline', 'tag', 'text', 'thumbtack-angle', 'ticket-wrench', 'ticket-wrench-multiple', 'triangle-down', 'triangle-left', 'triangle-right', 'triangle-up', 'triangle-wrench', 'unlock-outline', 'unlock', 'user-outline', 'user-settings', 'user', 'users', 'verified', 'video-camera-mute', 'video-camera', 'visible-off', 'visible', 'voip', 'warning-outline', 'warning', 'wifi-outline', 'wifi', 'wrench', 'x-alt', 'x']
```

Comment:
Could we use v-for here?

Reply:
I dont think it will be needed in this file and below files as there are chi components utilized within it.

Reply:
Having icons array defined in the $data would make the HTML / pug part of the component much more readable.

Reply:
Made the required changes and updated the PR.